macros/LearnSidebar.ejs

```diff
@@ -1357,265 +1357,329 @@ var text = mdn.localStringMap({
 
 <ol>
   <li data-default-state="<%=currentPageIsUnder('Getting_started_with_the_web')%>"><a href="<%=baseURL%>Getting_started_with_the_web"><strong><%=text['Complete_beginners']%></strong></a></li>
-  <li data-default-state="<%=currentPageIsUnder('Getting_started_with_the_web')%>"><a href="<%=baseURL%>Getting_started_with_the_web"><%=text['Getting_started_with_the_web']%></a>
-    <ol>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web"><%=text['Getting_started_with_the_web_overview']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/Installing_basic_software"><%=text['Installing_basic_software']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/What_will_your_website_look_like"><%=text['What_will_your_website_look_like']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/Dealing_with_files"><%=text['Dealing_with_files']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/HTML_basics"><%=text['HTML_basics']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/CSS_basics"><%=text['CSS_basics']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/JavaScript_basics"><%=text['JavaScript_basics']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/Publishing_your_website"><%=text['Publishing_your_website']%></a></li>
-      <li><a href="<%=baseURL%>Getting_started_with_the_web/How_the_Web_works"><%=text['How_the_Web_works']%></a></li>
-    </ol>
+  <li class="toggle">
```

Comment:
The ``{{LearnSidebar}}`` is huge, and appears to be correctly converted 👍. There are some elements with ``data-default-state="closed"`` and ``="open"``, but they do not appear to affect rendering on https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML:

<img width="641" alt="pr 789 learnsidebar" src="https://user-images.githubusercontent.com/286017/45386859-0b301a80-b5da-11e8-8794-74b9dc8472a3.png">